samples/java_springboot/18.bot-authentication/src/main/java/com/microsoft/bot/sample/authentication/AuthBot.java

```diff
@@ -16,6 +16,7 @@
 import com.codepoetics.protonpack.collectors.CompletableFutures;
 import com.microsoft.bot.schema.Activity;
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.LoggerFactory;
 
 public class AuthBot extends DialogBot<MainDialog> {
```

Comment:
Something I noticed now, this class in [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/18.bot-authentication/Bots/AuthBot.cs#L14) extends from generic type T where T is a Dialog, should we do the same that we did in the [DialogAndWelcomeBot](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/DialogAndWelcomeBot.java#L30) in the sample 21? Also in the Constructor the parameter dialog, should be generic. Check [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/18.bot-authentication/Bots/AuthBot.cs#L16).

Reply:
Fixed